src/components/Sidebar/menu.styles.js

```diff
@@ -0,0 +1,70 @@
+import {Button} from '@material-ui/core'
+import {makeStyles, withStyles} from '@material-ui/styles';
+export const BootstrapButton = withStyles({
+    root: {
```

Comment:
the function of material ui is makeStyles that function join the themplate with your components